app/views/fake/audio_segmenter.html.erb

```diff
@@ -0,0 +1,133 @@
+<% content_for :tabs do %>
+  <%= tab_link_to "Metadata", fake_path(123) %>
+  <%= tab_link_to "Audio Segmenter", fake_path("audio-segmenter") %>
+<% end %>
+
+<div data-controller="audio-breakpoints waveform-inspector"
+  data-audio-breakpoints-label-prefix-value="Mid Roll"
+  data-audio-breakpoints-segment-count-value="4"
+  data-audio-breakpoints-markers-value="[[148.723809],[153.29814058956916,157.19909297052155]]"
+  data-audio-breakpoints-target="waveformInspector"
+  data-waveform-inspector-playing-class="player--playing"
+  data-waveform-inspector-audio-url-value="/audio/TOL_6min_720p_download.mp3"
+  data-waveform-inspector-waveform-url-value="/audio/TOL_6min_720p_download.json"
+  data-action="resize@window->waveform-inspector#updateLayout audio-breakpoints:markers.update->waveform-inspector#updateMarkers waveform-inspector:marker.update->audio-breakpoints#updateBreakpointMarker audio-breakpoint:marker.update->audio-breakpoints#updateBreakpointMarker audio-breakpoint:play->waveform-inspector#playMarker audio-breakpoint:seekTo->waveform-inspector#seekToMarker">
+  <header class="prx-content-header | d-flex justify-content-end mx-4 px-2 py-3">
+    <h2 class="h1 fw-bold visually-hidden">Audio Segmenter</h2>
+    <div class="">
+      <button class="btn btn-primary">Save</button>
```

Comment:
At some point, we'll want to move any labels into `en.yml`.  But that can happen later.

Reply:
Yep. I just wasn't exactly sure how to do that. Would be happy to pair with whoever does the implementation of this into a working route.